server/etcdserver/apply.go

```diff
@@ -408,6 +408,9 @@ func (a *applierV3backend) Range(ctx context.Context, txn mvcc.TxnRead, r *pb.Ra
 		case r.SortTarget == pb.RangeRequest_VALUE:
 			sorter = &kvSortByValue{&kvSort{rr.KVs}}
 		}
+		if sorter == nil {
+			return nil, fmt.Errorf("sorter was nil")
```

Comment:
Please add the error to error.go file. 
I think that the error should point on "SortTarget" field, e.g. "etcdserver: badrequest: SortTarget not supported"